gamification/process/update.py

```diff
@@ -135,6 +136,27 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+def update_participants_data_with_meta_review(participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = participant.pos_in
+    pos_out_meta_review = participant.pos_out
+    neg_in_meta_review = participant.neg_in
+    neg_out_meta_review = participant.neg_out
+    if pos_in_meta_review > 0 or pos_out_meta_review > 0 or \
+       neg_in_meta_review > 0 or neg_out_meta_review > 0:
+        participant.meta_review_add_activity(participant)
```

Comment:
I don't think this would work as participant does not have `meta_review_add_activity` method.